rampwf/kits/ramp_kit.py

```diff
@@ -8,21 +8,22 @@
 
 BASE_RAMP_KIT_URL = 'https://github.com/ramp-kits/'
 
-RAMP_KITS_AVAILABLE = ('kaggle_seguro',
-                       'mars_craters',
-                       'MNIST_simplified',
-                       'MNIST',
-                       'california_rainfall_test',
-                       'boston_housing',
-                       'iris',
-                       'titanic',
-                       'epidemium2_cancer_mortality',
-                       'drug_spectra',
-                       'air_passengers',
-                       'el_nino',
-                       'HEP_tracking',
-                       'mouse_cytometry',
-                       )
+RAMP_KITS_AVAILABLE = []
+#   ('kaggle_seguro',
+#    #'mars_craters',
+#    'MNIST_simplified',
+#    'MNIST',
+#    'california_rainfall_test',
+#    #'boston_housing',
+#    #'iris',
+#    'titanic',
+#    'epidemium2_cancer_mortality',
+#    #'drug_spectra',
+#    #'air_passengers',
+#    #'el_nino',
+#    'HEP_tracking',
+#    'mouse_cytometry',
+#    )
```

Comment:
Cleanup?

Reply:
I think that we will just remove this module in fact